docs/meshstack.gcp.index.md

```diff
@@ -102,56 +102,14 @@ In order to associate created projects with a Billing Account, the replicator ne
 
 ### meshfed-service User
 
-The Service Account created above needs to impersonate a technical user to perform [delegated operations using the Admin SDK](https://developers.google.com/admin-sdk/directory/v1/guides/delegation).
-Depending on your organization's setup of Google Cloud Identity, provisioning a technical user in the cloud directory
-may require one of the following two alternatives.
+In order to perform certain group related administrative tasks the previosly created service account needs an additional role from the Admin Console (G Suite). When you have executed the Terraform Script to setup the GCP service account its unique id is printed after the creation.
```

Comment:
d: while we want to go in the direction of using terraform platform modules also in the docs, at the moment the doc instructions are all agnostic to how operators provision these things... 

I'd remove the reference for now until we have that sorted out

Reply:
rephrased it a bit and removed the sentence.